dev.ters.Chordcat.metainfo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.ters.Chordcat</id>
+  
+  <name>Chordcat</name>
+  <summary>Chord Naming App with MIDI Support </summary>
+  <developer id="dev.ters">
+    <name>Shriram Ravindranathan</name>
+  </developer>
+  
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <url type="homepage">https://shriramters.github.io/chordcat/</url>
+  <url type="bugtracker">https://github.com/shriramters/chordcat/issues</url>
+  <url type="contribute">https://github.com/shriramters/chordcat/</url>
+  <url type="help">https://shriramters.github.io/chordcat/</url>
+  
+  <description>
+    <p>
+      Chordcat is a desktop app that helps you identify and name chords based on inputted notes.
+    </p>
+    <p>
+      Features:
+    </p>
+    <ul>
+      <li>
+        Displays chord names when:
+        <ul>
```

Comment:
`ul` inside `li` isn't allowed. it will fail validation.

Reply:
Thanks! I have updated that now